Documentation/contributing/release/backports.rst

```diff
@@ -76,10 +76,29 @@ process for backporting these PRs:
 One-time Setup
 ~~~~~~~~~~~~~~
 
+#. Make sure you have a GitHub developer access token with the ``public_repos``
+   ``workflow`` scopes available. You can do this directly from
+   https://github.com/settings/tokens or by opening GitHub and then navigating
+   to: User Profile -> Settings -> Developer Settings -> Personal access token
+   -> Generate new token.
+
 #. The scripts referred to below need to be run on Linux, they do not work on
-   macOS. It is recommended to use the dev VM (``contrib/vagrant/start.sh``),
-   as it will have all the correct versions of dependencies / libraries. Once
-   you have a machine ready, you need to configure git to have your name and
+   macOS. It is recommended to create a container using (``contrib/backporting/Dockerfile``),
+   as it will have all the correct versions of dependencies / libraries. In order to
+   create a container based setup, build and run a container:
+
+   .. code-block:: shell-session
+
+      $ docker-compose -f contrib/backporting/docker-compose.yaml build
+      $ docker-compose -f contrib/backporting/docker-compose.yaml run -e GITHUB_TOKEN=<YOUR_TOKEN> backport /bin/bash
```

Comment:
Do you know if there is a way with docker-compose to pass additional environment variables at build time (or if not, at run time) to pass the git username and email? If we could copy-paste a command like:
```
$ docker-compose -f ... -e GITHUB_USERNAME=$(git config user.name) -e <same-for-email> build
```
this would avoid having to set the parameters manually in the container?

(Could be a follow-up to this PR, if doable at all.)

Reply:
That's certainly doable (that's how users would pass their GitHub token), but I'm not sure if that's required here. We volume mount user's git repo inside the container, and the users still have to follow these instructions given in the guide once.